CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## TBD
+
+- Add support for `TroposphereType`
```

Comment:
I'd just move this into the 1.0.0a1 section - I plan to just update that to be the 1.0.0 release when we're ready to go.